tests/functions/test-wpseo-xml-sitemaps-base-url.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
I'm missing the test that possibly causes this issue according to #2616: 
> Could this cause the issue when ran on the backend (on https) instead of the frontend (on http)?